apps/codeforafrica/src/payload/fields/impacts.js

```diff
@@ -0,0 +1,14 @@
+import { deepmerge } from "@mui/utils";
+
+const impacts = (overrides) =>
+  deepmerge(
```

Comment:
Where is this being used?

Reply:
Then you need to make a call... and something on this PR that uses this field (e.g. GetInvolved block) or move this field to one of those PRs.